src/DevSoc.Eventathon.Data/Migrations/Script0002-Attendance.sql

```diff
@@ -0,0 +1,5 @@
+﻿CREATE TABLE Attendance (
```

Comment:
This script needs to be an `EmbeddedResource` and should copy to output if newer
(Right click > Properties to set this in Rider)

Reply:
FYI this is why it doesn't migrate automatically